Microsoft.Toolkit.Uwp.SampleApp/SamplePages/AdaptiveGridView/AdaptiveGridViewPage.xaml.cs

```diff
@@ -42,14 +43,14 @@ public AdaptiveGridViewPage()
             AdaptiveGridViewControl.SelectionChanged += AdaptiveGridViewControl_SelectionChanged;
         }
 
-        private void AdaptiveGridViewControl_SelectionChanged(object sender, Windows.UI.Xaml.Controls.SelectionChangedEventArgs e)
+        private void AdaptiveGridViewControl_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
             SelectedItemCountTextBlock.Text = AdaptiveGridViewControl.SelectedItems.Any()
                 ? $"You have selected {AdaptiveGridViewControl.SelectedItems.Count} items."
                 : "You haven't selected any items";
         }
 
-        private async void AdaptiveGridViewControl_ItemClick(object sender, Windows.UI.Xaml.Controls.ItemClickEventArgs e)
+        private async void AdaptiveGridViewControl_ItemClick(object sender, ItemClickEventArgs e)
```

Comment:
Not an error but some plugin you have. You can tell the namespace isn't an issue b/c said plugin added the using statement (just my guess)